### PR TITLE
ComboBox: Adding support for Alt + Up to close menus

### PR DESCRIPTION
**Pull request by chang47**
**Problem:**
We want to be able to close menu's by using Alt + up to be consistent with the behavior of how Office applications interact with menu's.

Currently, we don't have this behavior with our menu items. Specifically, if we were to use Alt + Down on a button/combo box to expand the menu, we should be able to use Alt + Up to collapse the menu.

**Solution:**
I added another condition for when we want to close a menu that would check for Alt + Up.

**Validation:**
I validated that this new keyboard commands will now close menu's in both combo boxes and anything that uses the contextual menu's specifically buttons

Specifically, the problem case that would show up is if we open the menu with alt and then while still holding down the alt button, if we hit up arrow, we want to be able to close the menu.

### CODE REVIEW

**Comment by chang47** [comment]
@jspurlin

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx [hunk 1]
```diff
@@ -694,7 +694,8 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
   private _onKeyDown = (ev: React.KeyboardEvent<HTMLElement>) => {
     if (ev.which === KeyCodes.escape ||
-      ev.altKey ||
+      ev.which === KeyCodes.alt ||
+      (ev.which === KeyCodes.up && ev.altKey) ||
```
Comment (jspurlin) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx]: In my testing contextual menus already collapse on ALT + UpArrow... Are changes to this file needed?

Comment (chang47) [reply packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx]: You are correct. I went and confirmed that. I'll remove this, but it looks like the changes for the combo box are still needed.

File: packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx [hunk 2]
```diff
@@ -1464,6 +1464,11 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
           index = this.state.currentOptions.length;
         }
 
+        if (ev.altKey && isOpen) {
```
Comment (jspurlin) [inline packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx]: I think we also want meta key to work here (see line 1478, for example)

File: common/changes/office-ui-fabric-react/altMenu_2018-04-04-18-25.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "ComboBox: Adding support for Ctrl + Up to close menus",
```
Comment (jspurlin) [inline common/changes/office-ui-fabric-react/altMenu_2018-04-04-18-25.json]: It's ALT (or Meta), not CTRL (+ UpArrow) that you adding support for

**Pull request merged**